src/diffusers/models/vae.py

```diff
@@ -171,7 +173,7 @@ def __init__(
 
             up_block = get_up_block(
                 up_block_type,
-                num_layers=self.layers_per_block + 1,
```

Comment:
This breaks stable diffusion

Reply:
Yes I know, so we'll have to add more if-else statements, making the code less readable etc.

Reply:
no, we just add a `decoder_layers_per_block` config attribute that defaults to `None` which then defaults to `layers_per_block + 1` which you can overwrite